packages/mobile-top-nav/src/assets/img/icon-ellipses-static.js

```diff
@@ -0,0 +1,19 @@
+import { html } from 'lit-element';
+
+export default html`
```

Comment:
What is the difference between the static version of these icons and the non-static?

Reply:
I left them in for you to review to see if the static conversion was what you were talking about. At this point the non-static ones can be removed and these renamed.

Reply:
Ah yeah this is what I had in mind!